db/db.js

```diff
@@ -120,31 +122,44 @@ export async function initializeDatabase() {
         UserID INT NOT NULL,
         Play BOOLEAN NOT NULL default 0,
         VoteType ENUM('upvote', 'downvote') NOT NULL,
-        UNIQUE (TrackID, UserID),
         FOREIGN KEY (UserID) REFERENCES users(id) ON DELETE CASCADE,
         FOREIGN KEY (TrackID) REFERENCES tracks(id) ON DELETE CASCADE
       )
     `);
 
     await conn.query(`
       CREATE TABLE IF NOT EXISTS playlists (
-        id INT AUTO_INCREMENT PRIMARY KEY,
+        PlaylistID INT AUTO_INCREMENT PRIMARY KEY,
         name VARCHAR(255) NOT NULL,
+        Description TEXT DEFAULT NULL,
         user_id INT NOT NULL,
         created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
```

Comment:
Ensure that styles align, will have to update this in a later PR to utilize camel case instead of snake case